index.js

```diff
@@ -8,14 +8,24 @@ const clone = require('lodash.clonedeep');
 const uuid = require('uuid/v4');
 
 /**
+ * Proxy polyfill injection.
+ */
+
+require('proxy-polyfill');
```

Comment:
This pollutes the global namespace

<img width="418" alt="screen shot 2017-01-10 at 11 49 28" src="https://cloud.githubusercontent.com/assets/257346/21805209/ec148bd4-d72a-11e6-8038-d83550799544.png">